src/main/java/de/imi/mopat/controller/ClinicController.java

```diff
@@ -412,4 +494,84 @@ public String removeClinic(@RequestParam(value = "id", required = true) final Lo
         }
         return showClinics(model);
     }
+
+    private List<ClinicConfigurationMapping> processClinicConfigurationMappingDTOS(Clinic clinic,
+        final List<ClinicConfigurationMappingDTO> clinicConfigurationMappingDTOS) {
+        List<ClinicConfigurationMapping> clinicConfigurationMappingList = new ArrayList<>();
+        for (ClinicConfigurationMappingDTO clinicConfigurationMappingDTO : clinicConfigurationMappingDTOS) {
+            ClinicConfiguration clinicConfiguration = clinicConfigurationDao.getElementById(
+                clinicConfigurationMappingDTO.getClinicConfigurationId());
+            if (clinic.getId() == null || clinic.getClinicConfigurationMappings().isEmpty()) {
+                ClinicConfigurationMapping clinicConfigurationMapping = new ClinicConfigurationMapping(clinic,
```

Comment:
Example for single argument line breaks

Either open bracket and then move every argument in a new line or move everything after the "=" to a new line to keep the expression in a single line